documentation/ci-fix.md

```diff
@@ -263,6 +265,25 @@ that the generated OpenAPI spec files were not in-sync with the TypeSpec project
 
 If none of the above helped, please reach out on [TypeSpec Discussions Teams channel].
 
+## APIView Failures: troubleshooting guides
+
+### Passed build but APIView not generated
+
+- First, ensure that all checks in the CI are green before proceeding. 
+- You should also see the comment box pictured below with a reference to your language:
+
+![alt text](image-3.png)
+
+
+If the above requirements are met proceed with the following steps, in order:
+
+1. Check for an unexpected skip of `Publish SDK APIView Artifact to Pipeline Artifacts` and `Generate SDK APIView` step.
+2. Look in `SDK Automation` step to verify that the API token generation completed successfully.
+3. Search logs for `Read Temp File`
+4. Below `Read Temp File` find the .json object and search within to locate the `apiViewArtifact` property.
+5. If not present, the APIView parser for the language failed to generate the `APIView Token Artifacts`.
```

Comment:
I think you are mixing here two different steps and set of instructions. Let's talk offline and get this clarified